tests/integration/test_diagnostics_mixin.py

```diff
@@ -0,0 +1,46 @@
+""" Tests of functionality of porepy.vis.diagnostics_mixin.DiagnosticsMixin.
```

Comment:
Placement of this file: The other model-related mixins are located in tests/integration/models. I suggest moving this as well.